include/unifex/sender_concepts.hpp

```diff
@@ -179,14 +179,9 @@ UNIFEX_CONCEPT //
     detail::_has_sender_types<sender_traits<remove_cvref_t<S>>>;
 
 template <typename S>
-UNIFEX_CONCEPT   //
-  typed_sender = //
-    sender<S>;
-
-template <typename S>
-UNIFEX_CONCEPT        //
-  typed_bulk_sender = //
```

Comment:
As above with `typed_sender`, maybe this should be retained as a deprecated alias for `bulk_sender`.